volto/src/customizations/components/theme/App/App.css

```diff
@@ -0,0 +1,15 @@
+main {
+  display: block;
+}
+.content-area {
+  min-height: 100%;
+  margin: 0 32px;
+  display: grid;
+  grid-template-columns: 256px auto;
```

Comment:
So this looks fine and 256px is what Gareth uses in climate-change.data.gov.uk, but then the gap between the nav and content (`div#main / .view`) is too big and I can't figure out what's doing it. Let's raise a separate issue to investigate.